sematic/resolvers/cloud_resolver.py

```diff
@@ -221,7 +222,7 @@ def _wait_for_any_remote_job(self) -> Optional[str]:
                     self._refresh_graph(future.id)
                     return future.id
 
-            logger.info("Sleeping for %s s", delay_between_updates)
+            logger.debug("Sleeping for %s s", delay_between_updates)
```

Comment:
Same point: users don't see these logs (unless they do non-detach), but it is valuable for us to have. "Is this pod that looks stuck actually doing something? Is it waiting for some event that has already happened? Or is the program spinning in an infinite loop? This failure mode only happens once every 1000 runs in this very specific context, so it would be nice if info was available to help diagnose!"